low-level/files/etc/rsyslog.d/0-iemfwd.conf

```diff
@@ -1,30 +0,0 @@
-# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
```

Comment:
This file is needed for journal to file IEM forwarding, IEM sensor will not work without this.
This file is not related to SSPL logs.

Reply:
Ok, thanks for identifying. we should keep it then

Reply:
Addressed, please check.